tasks/app/initialize.py

```diff
@@ -86,37 +86,19 @@ def initialize_gitlab_submissions(context, email, dryrun=False):
     ]
     submission_data = [
         (uuid.UUID('00000000-0000-0000-0000-000000000001'), []),
-        (uuid.UUID('00000000-0000-0000-0000-000000000002'), image_data),
+        (uuid.UUID('00000000-0000-0000-0000-000000000003'), image_data),
```

Comment:
No this was intentional. 
002 on gitlab had the files in the _submissions dir:
https://sub.dyn.wildme.io/test/00000000-0000-0000-0000-000000000002
which are symlinked from the _assets dir
I debated tweaking 002 so that it had both _submissions and _asset_group but decided that a new one with the new format was the best option.
https://sub.dyn.wildme.io/test/00000000-0000-0000-0000-000000000003